testsuite/shared/src/main/java/org/jboss/as/test/shared/LayersTestBase.java

```diff
@@ -200,9 +200,6 @@ public abstract class LayersTestBase {
             "org.eclipse.microprofile.lra.api",
             // Extension not included in the default config
             "org.wildfly.extension.microprofile.openapi-smallrye",
-            "org.eclipse.microprofile.openapi.api",
-            "io.smallrye.openapi",
-            "com.fasterxml.jackson.dataformat.jackson-dataformat-yaml",
```

Comment:
Why are these now referenced?

Reply:
Because of this additional optional dependency: https://github.com/pferraro/wildfly/blob/openapi/ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/org/wildfly/extension/undertow/main/module.xml#L47